packages/p2p/src/stores/my-profile-store.js

```diff
@@ -524,7 +524,7 @@ export default class MyProfileStore extends BaseStore {
             this.setIsConfirmDeleteModalOpen(true);
         }
     }
-  
+
```

Comment:
@yauheni-kryzhyk-deriv do we need to change it in this PR?😊

Reply:
Don’t understand this issue, when I delete this space it shows me this deleting like my changes. If I add it, it shows me, that it is also my changes. will leave it with space as it is the same in the code